docs/quickstart/README.md

```diff
@@ -0,0 +1,198 @@
+# SRF Quick Start Guide (QSG)
+
+The SRF Quick Start Guide (QSG) provides examples on how to start using SRF via the Python bindings, C++ bindings or both.
+
+## Prerequisites
+
+- Pascal architecture or better
```

Comment:
Do we also need to mention there's a CUDA compute capability requirement?

Reply:
We can but dont they go hand in hand? What is our compute capability requirement?

Reply:
Added 6.0 or higher in the requirements